bucket/wizmouse.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "1.7.0.3",
+    "description": "Allows mouse wheel scrolling for the window under the pointer instead of the currently focused window.",
+    "homepage": "https://antibody-software.com/web/software/software/wizmouse-makes-your-mouse-wheel-work-on-the-window-under-the-mouse/",
+    "license": "Unknown",
+    "url": "https://antibody-software.com/files/wizmouse_1_7_0_3_portable.zip",
+    "hash": "452e75ef27db171e1bca55fc1762a1736da98b087eb12803a21de4c925edfbbd",
+    "pre_install": "if (-not (Test-Path \"$persist_dir\\wizmouse.ini\")) { New-Item \"$dir\\wizmouse.ini\" -ItemType File | Out-Null }",
+    "bin": [
+        "WizMouse.exe",
+        "WizMouseConfig.exe"
+    ],
+    "shortcuts": [
+        [
+            "WizMouse.exe",
+            "WizMouse"
+        ],
+        [
+            "WizMouseConfig.exe",
+            "WizMouseConfig"
+        ]
+    ],
+    "persist": "wizmouse.ini",
+    "checkver": "v([\\d.]+)",
```

Comment:
```suggestion
    "checkver": "([\\d.]+)\\?",
```

Reply:
.

Reply:
Wouldn't `ver ([\\d.]+)` be better since it'll only match the top number

Reply:
No